public/locales/fr/common.json

```
@@ -9,19 +9,19 @@
   "albumArtist": "Artiste de l'album",
   "history": "Historique",
   "yourProfile": "Votre profil",
-  "yourHistory": "Your history",
-  "scrobbleManually": "Scrobble manually",
-  "scrobbleFromOtherUser": "Scrobble from other user",
-  "otherUser": "User",
+  "yourHistory": "Votre historique",
+  "scrobbleManually": "Scrobbler manuellement",
+  "scrobbleFromOtherUser": "Scrobbler depuis un autre utilisateur",
+  "otherUser": "Utilisateur",
   "or": "ou",
   "refresh": "Actualiser",
-  "findFriendCopy": "Enter a last.fm username to see their last tracks",
-  "visitProfile": "View profile",
-  "search": "Search",
-  "backToSearch": "Back to search",
-  "userNotFound": "User not found",
-  "invalidUsername": "Invalid username",
-  "recentlySearchedUsers": "Searched recently",
+  "findFriendCopy": "Entrez un nom d'utilisateur last.fm pour voir leurs dernières pistes",
+  "visitProfile": "Voir le profil",
+  "search": "Rechercher",
+  "backToSearch": "Retourner sur la recherche",
+  "userNotFound": "Aucun utilisateur trouvé",
+  "invalidUsername": "Nom d'utilisation invalide",
+  "recentlySearchedUsers": "Récement recherche",
```

This is for recently searched (past tense) users, is this conjugation correct?

```suggestion
  "recentlySearchedUsers": "Récemment recherchés",
```

---

My bad :x 

Yes That'.s right, that's right! I wanted to correct this mistake for `récemment` but I PR without thinking about it again. For `recherchés` this also needs to be corrected. I still have no idea of the conjugation and time.

---

Don't worry, the code reviews are perfect to iron out these details 😃